en/micro-integrator/docs/references/mediators/property-reference/accessing-properties-with-xpath.md

```diff
@@ -1,6 +1,14 @@
 # Accessing Properties with XPath
 
-The WSO2 Micro Integrator supports standard XPath functions and variables through its underlying XPath engine. The Micro Integrator also provides custom XPath functions and variables for accessing message properties.
+The WSO2 Micro Integrator supports standard XPath functions and variables through its underlying XPath engine. It supports 
+XPath 1.0 by default where as the support for XPath 2.0 can be introduced by adding the following property in <MI_HOME>/conf/deployment.toml.
```

Comment:
WSO2 Micro Integrator supports standard XPath functions and variables through its underlying XPath engine. It supports XPath 1.0 by default where as XPath 2.0 is supported if you add the following property to the `deployment.toml` (stored in the `<MI_HOME>/conf` folder).